### ISSUE TITLE
[Bug] Missing type definition in puppeteer-extra@3.3.1

### ISSUE THREAD

**Issue by wxh06** [berstend/puppeteer-extra#671]
### Describe the bug

I think it is a packaging problem.

#### puppeteer-extra@3.3.1 (error occurs)

```
node_modules/
    puppeteer-extra/
        dist/
            index.cjs.js
            index.cjs.js.map
            index.d.ts
            index.esm.js
            index.esm.js.map
            index.js
            index.js.map
        LICENSE
        package.json
        readme.md
```

In `node_modules/puppeteer-extra/dist/index.d.ts`:

```typescript
/// <reference path="../src/puppeteer-legacy.d.ts" />
import { PuppeteerNode } from 'puppeteer';
```

First, it should be `../dist/puppeteer-legacy.d.ts` instead of `../src/puppeteer-legacy.d.ts`.
Second, `ambient.d.ts` and `puppeteer-legacy.d.ts` are missing in this version.

<details>
<summary>

#### puppeteer-extra@3.3.0 (works properly)

</summary>

```
node_modules/
    puppeteer-extra/
        dist/
            ambient.d.ts
            index.cjs.js
            index.cjs.js.map
            index.d.ts
            index.esm.js
            index.esm.js.map
            index.js
            index.js.map
            puppeteer-legacy.d.ts
```

In `node_modules/puppeteer-extra/dist/index.d.ts`:

```typescript
/// <reference path="../dist/puppeteer-legacy.d.ts" />
import { PuppeteerNode } from 'puppeteer';
```

</details>

### Code Snippet

```typescript
import puppeteer from 'puppeteer-extra';
```

```
node_modules/puppeteer-extra/dist/index.d.ts:2:10 - error TS2305: Module '"puppeteer"' has no exported member 'PuppeteerNode'.

2 import { PuppeteerNode } from 'puppeteer';
           ~~~~~~~~~~~~~

node_modules/puppeteer-extra/dist/index.d.ts:1:22 - error TS6053: File '/workspace/node_modules/puppeteer-extra/src/puppeteer-legacy.d.ts' not found.

1 /// <reference path="../src/puppeteer-legacy.d.ts" />
                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

### Versions

This issue was introduced in the most recent version puppeteer-extra@3.3.1.

```
  System:
    OS: Linux 5.10 Debian GNU/Linux 11 (bullseye) 11 (bullseye)
    CPU: (4) arm64 unknown
    Memory: 1.81 GB / 3.84 GB
    Container: Yes
    Shell: 5.1.4 - /bin/bash
  Binaries:
    Node: 16.15.1 - /usr/local/bin/node
    Yarn: 3.0.2 - /usr/local/share/npm-global/bin/yarn
    npm: 8.11.0 - /usr/local/bin/npm
```

**Comment by layerlre** [comment]
same here
`node_modules/puppeteer-extra/dist/index.d.ts:1:22 - error TS6053: File './node_modules/puppeteer-extra/src/puppeteer-legacy.d.ts' not found.

1 /// <reference path="../src/puppeteer-legacy.d.ts" />
                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
`

**Comment by Eunsub-thumb** [comment]
`node_modules/puppeteer-extra/src/puppeteer-legacy.d.ts' not found.`

I got same issue

**Comment by berstend** [comment]
Looking into it :)

**Comment by dupocas** [comment]
For now downgrading to `3.3.0` fixes the issue

**Comment by berstend** [comment]
Fixed in `puppeteer-extra@3.3.2`, apologies 😄
